src/js/events/MarathonActions.js

```diff
@@ -495,6 +497,48 @@ var MarathonActions = {
         });
       }
     });
+  },
+
+  killPodInstances(pod, instanceIDs, force) {
+    let podID = encodeURIComponent(pod.getId());
+    let params = '';
+
+    if (!(pod instanceof Pod)) {
```

Comment:
Hey @wavesoft I like that you're thinking about these checks but do you really think we need them here? I think we should bring this up in a team sync to chat about it but for now we should remove them for consistency.